### PR TITLE
chore: release 2.23.2

### PR DESCRIPTION
**Pull request by arsenijesavic**
Release-As: 2.23.2

### CODE REVIEW

File: .github/workflows/deploy-prod.yml [hunk 1]
```diff
@@ -32,8 +32,8 @@ jobs:
           BLOCKCHAIN_EXPLORER_BTC: 'https://www.blockchain.com/btc/tx/'
           BLOCKCHAIN_EXPLORER_ETH: 'https://etherscan.io/tx/'
           BLOCKCHAIN_EXPLORER_EOS: 'https://bloks.io/transaction/'
-          PPP_ENV: 'prod'
-          SENTRY_DSN: 'https://0b768263dc6247a5a2d309e07f771528@o324628.ingest.sentry.io/4503898341703680'
+          PPP_ENV: 'eos'
```
Comment (alex-enginelab) [inline .github/workflows/deploy-prod.yml:35]: @arsenijesavic This is not good. On prod it should be `prod`. Actually best would be to add `${{vars.pppEnv}}` or something like that

File: .github/workflows/deploy-prod.yml [hunk 2]
```diff
@@ -32,8 +32,8 @@ jobs:
...
+          SENTRY_DSN: 'https://6fa219bfb2ed453fb5032d6ce1b83fc9@sentry.io/1884471'
```
Comment (alex-enginelab) [inline .github/workflows/deploy-prod.yml:36]: @arsenijesavic Please, can you add this also as `${{vars.sentryDNS}}` Thanks

File: .github/workflows/deploy-staging.yml [hunk 3]
```diff
@@ -33,7 +33,7 @@ jobs:
           BLOCKCHAIN_EXPLORER_ETH: 'https://etherscan.io/tx/'
           BLOCKCHAIN_EXPLORER_EOS: 'https://bloks.io/transaction/'
           PPP_ENV: 'prod'
-          SENTRY_DSN: 'https://0b768263dc6247a5a2d309e07f771528@o324628.ingest.sentry.io/4503898341703680'
+          SENTRY_DSN: 'https://6fa219bfb2ed453fb5032d6ce1b83fc9@sentry.io/1884471'
```
Comment (alex-enginelab) [inline .github/workflows/deploy-staging.yml:36]: @arsenijesavic If you can do same as in the previous file. Thanks

File: package.json [hunk 4]
```diff
@@ -25,7 +25,7 @@
     "apollo-utilities": "^1.3.4",
     "@sentry/browser": "6.3.1",
     "@sentry/integrations": "6.3.1",
-    "@smontero/ppp-client-api": "0.0.43",
```
Comment (alex-enginelab) [inline package.json:28]: @arsenijesavic Please use `@smontero/ppp-client-api` for now. I need to test and ensure that `@hypha-dao/ppp-client-api` is working correctly

File: src/boot/ppp.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import PPP from '@smontero/ppp-client-api'
```
Comment (alex-enginelab) [inline src/boot/ppp.js:1]: @arsenijesavic Please use `@smontero/ppp-client-api` for now

**Review by alex-enginelab — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy-prod.yml (hunk 1 above)
Comment (Alex-IS) [inline .github/workflows/deploy-prod.yml:35]: @arsenijesavic Please set this as `${{ vars.PPP_ENV }}`

File: .github/workflows/deploy-prod.yml (hunk 2 above)
Comment (Alex-IS) [inline .github/workflows/deploy-prod.yml:36]: @arsenijesavic please set this as `${{ vars.SENTRY_DSN }}`

**Review by Alex-IS — CHANGES_REQUESTED** [review]

**Review by Alex-IS — APPROVED** [review]
👍

**Pull request merged**